src/AccessLogEntry.h

```diff
@@ -60,8 +60,8 @@ class AccessLogEntry: public CodeContext
     /// Side effect: Enables reverse DNS lookups of future client addresses.
     const char *getLogClientFqdn(char *buf, size_t bufSize) const;
 
-    /// Fetch the client IDENT string, or nil if none is available.
-    const char *getClientIdent() const;
+    /// \returns RFC 1413 user identity (including empty strings), if any
```

Comment:
Just to double check: Here, "empty string" refers to a special Ident::User value that indicates an Ident failure (including but not limited to an empty user-id value received from the Ident server), right?

Reply:
Yes, the "empty string" indicates that a started Ident job could not obtain a non-empty ident string (due to a error, e.g., invalid input).